### PR TITLE
Remove common import creds workflow

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR removes `workflows/common-import-credentials.yml`, as discussed in https://github.com/w3c-ccg/traceability-vocab/issues/887

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]
This is fine overall, Nis will be rebasing this PR to fix the extraneous addition of the apparel HTML

File: docs/sections/workflows.html [hunk 1]
```diff
@@ -34,6 +34,250 @@ <h3>Business Card Workflow</h3>
       A->>B: present Verifiable Business Card</pre>
     
 
+      <h3>CBP E-Commerce Apparel Tech Demo 2024</h3>
```
Comment (nissimsan) [inline docs/sections/workflows.html:37]: This should not be part of this PR.

**Review by nissimsan — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

File: docs/sections/workflows.html (hunk 1 above)
Comment (nissimsan) [reply docs/sections/workflows.html:37]: Understood, this was never build into main (only gh pages). Fine then.
